qt/app.py

```diff
@@ -139,7 +139,7 @@ def __init__(self, config, appInstance, qapp):
         # right widget
         self.filesWidget = QGroupBox(self)
         filesLayout = QVBoxLayout(self.filesWidget)
-        left, top, right, bottom = filesLayout.getContentsMargins()
+        right= filesLayout.getContentsMargins()
```

Comment:
Do you know about [tuple unpacking](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences)?

Suppose that `fileLayout.getContentMargins()` returns a tuple of values such as `(1, 2, 3, 4)`. In the original code, these are unpacked as follows:

```
left = 1
top = 2
right = 3
bottom = 4
```

But in your change, you assign the whole tuple to `right`:

```
right = (1, 2, 3, 4)
```

I can't imagine that this will work.
If you want to remove the unused variables without breaking the code, you probably need to set `right` to the third value from the tuple:

```
right = filesLayout.getContentsMargins()[2]
```

Note, I see you have done the same thing in many places below.

Reply:
Thank you for your review. Yes, I am aware of the tuple unpacking and I am working on fixing those mistakes.